assets/javascripts/initializers/discourse-calendar.js

```diff
@@ -36,14 +39,21 @@ function initializeDiscourseCalendar(api) {
 
   api.onPageChange((url) => {
     const $calendarContainer = $(`${selector}.category-calendar`);
-    if (!$calendarContainer.length) {
-      return;
+    if ($calendarContainer.length) {
+      $calendarContainer.hide();
```

Comment:
I know you didn't add this, but we should be moving away from jQuery. Maybe a first step here can be to avoid using jQuery's hide/show. (Maybe add/remove the `.hidden` class instead?)

Also, if we could strip out some jquery, that would be great too.